include/tables/bombers_notebook/event_table.h

```diff
@@ -0,0 +1,46 @@
+/**
+ * Bombers Notebook Event Table
+ *
+ * DEFINE_EVENT should be used for events in the notebook
+ *    - Argument 0: Enum value for the event (without the pfefix)
```

Comment:
```suggestion
 *    - Argument 0: Enum value for the event (without the prefix)
```
imo without the prefix makes searching a bit more annoying, because for `BOMBERS_NOTEBOOK_EVENT_RECEIVED_ROOM_KEY`, you wouldn't find the source immediately, then you'd have to search the substring `EVENT_RECEIVED_ROOM_KEY` which picks up a few other defines as well as this one.

I can see why it's done for the `DEFINE_PERSON` though since you're trying to define two enums with 1 name. But then should `BombersNotebookPerson` be a different enum to `BombersNotebookEvent`? (I haven't looked through the system yet so maybe it will answer itself).

edit: coming back to this later, you could also drop the `WEEKEVENTREG_` from the `WEEKEVENTREG_BOMBERS_NOTEBOOK_EVENT_RECEIVED_ROOM_KEY` for both `event_table.h` and `person_table.h` and reuse that define for both `WEEKEVENTREG` and the enum introduced in the message branch `BombersNotebookEvent` (see https://github.com/Decompollaborate/mm/commit/c76ad75e42ee4da5f0dc4b6fcc390593563365b8)
For that reason, should the contents of Zoey's PR be included in this PR since they interlink? (i.e. add in that enum, name `D_801C6AB8`, and the docs for the two `BombersNotebookQueueEvent` functions)?

Reply:
This would probably work better in Discord for more visibility, so will respond there.
(Small note I did have a version that used the full name in this commit https://github.com/zeldaret/mm/pull/1235/commits/13b1f9c4364d468ad13c4860b0aaa7a98024c7f8, but it seemed really long so tried token pasting in https://github.com/zeldaret/mm/pull/1235/commits/13b1f9c4364d468ad13c4860b0aaa7a98024c7f8 and liked it better)

Reply:
Ended up going with just the full enum names again, as well as bring over Zoey's work from the message branch.